### PR TITLE
!!! Add `FieldComponent`  and add `FieldContainer`

### PR DESCRIPTION
**Pull request by mficzel**
FEATURE: Add `FieldContainer` prototype that renders label and validation input around the given afx content.

The `FieldContainer` will instantiate the given field and render a container with a label and error messages around the afx content.  It does not render the actual inputs. Those are rendered by prototypes derived from `FieldComponent`.

```
<Neos.Fusion.Form:FieldContainer
property="example"
label="Some Checkbox"
>
    <Neos.Fusion.Form:Checkbox value="1" checked />
    <Neos.Fusion.Form:Checkbox value="2" />
    <Neos.Fusion.Form:Checkbox value="3" />
</Neos.Fusion.Form:FieldContainer>
```

The `Neos.Fusion:Form:FieldComponent` is a base prototype for the field related html fragments it supports `id`, `class`, `required`, `attributes`. The purpose of the `FieldComponent` is to render the actual html-control. It uses existing `field` definitions from the context but will create a local one if `name` or `property` are found so those elements can be used directly without FieldContainer.

Other changes:
- Add general options `id`,`class`,`required` for all concrete field types
- Remove strict mode from propTypes
- Remove `*.Multiple` prototyes in favor of a `multiple` option for checkbox and select
- Rename `targetUri` to `action`
- Fixed rendering of referrers 
- Fixed extracting of field validationResults from from mappingResults

### CODE REVIEW

File: Resources/Private/Fusion/Prototypes/Checkbox.fusion [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
+
+    @propTypes {
+        multiple = ${PropTypes.boolean}
+        checked = ${PropTypes.boolean}
+    }
+
+    multiple = false
+    checked = false
+    # override checked state for object/submitted values
+    checked.@process.single = ${field.current == field.value}
```
Comment (skurfuerst) [inline Resources/Private/Fusion/Prototypes/Checkbox.fusion]: hm, do we want to name the properties "currentValue" and "initialValue" maybe?

Comment (skurfuerst) [reply Resources/Private/Fusion/Prototypes/Checkbox.fusion]: hm... I still struggle with finding good names...

File: Resources/Private/Fusion/Prototypes/Field.fusion [hunk 2]
```diff
@@ -2,18 +2,35 @@ prototype(Neos.Fusion.Form:Field)  < prototype(Neos.Fusion:Component) {
     @context.field = ${Form.createFieldDefinition(this.form, this.name, this.property, this.value)}
 
     @propTypes {
-        @strict = true
```
Comment (skurfuerst) [inline Resources/Private/Fusion/Prototypes/Field.fusion]: why did you remove strictness? IMHO enabling this leads to better error messages :)

File: Resources/Private/Fusion/Prototypes/Select.fusion [hunk 3]
```diff
@@ -1,21 +1,36 @@
...
-        renderer.attributes.selected = ${(field && (Form.stringifyValue(field.value) == Form.stringifyValue(props.value))) ? true : false}
+        # override selected for object/submitted values
+        renderer.attributes.selected.@process.singlevalue = ${(field.current == props.value) ? true : false}
+        renderer.attributes.selected.@process.singlevalue.@if.hasCurrent = ${field.current}
+        renderer.attributes.selected.@process.singlevalue.@if.isSingle = ${!__multiple}
+
+        # override selected for object/submitted multivalues
+        renderer.attributes.selected.@process.multivalue = ${Array.indexOf(field.current, props.value) > -1}
+        renderer.attributes.selected.@process.multivalue.@if.hasCurrent = ${field.current}
+        renderer.attributes.selected.@process.multivalue.@if.isMultiple = ${__multiple}
```
Comment (skurfuerst) [inline Resources/Private/Fusion/Prototypes/Select.fusion]: should we maybe write the Single/Multivalue distinction as Neos.Fusion:Case instead of "abusing" processors here?

File: Resources/Private/Fusion/Prototypes/Field.fusion (hunk 2 above)
Comment (mficzel) [reply Resources/Private/Fusion/Prototypes/Field.fusion]: strict causes hard to get errors when inherited prototypes add more props

File: Resources/Private/Fusion/Prototypes/Checkbox.fusion (hunk 1 above)
Comment (mficzel) [reply Resources/Private/Fusion/Prototypes/Checkbox.fusion]: removed the given value from afx entirely now, `field`-context only deals with bound or submitted values. If a concrete fields has a value prop it is fine to use it.

File: Resources/Private/Fusion/Prototypes/Select.fusion (hunk 3 above)
Comment (mficzel) [reply Resources/Private/Fusion/Prototypes/Select.fusion]: Switched to wrapper components now
